websites/A/AnimeVost/metadata.json

```diff
@@ -0,0 +1,66 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "296716315547402240",
+		"name": "ZebroUnion#2632"
+	},
+	"service": "AnimeVost",
+	"altnames": [
+		"АнимеВост"
+	],
+	"description": {
+		"en": "AnimeVost - Base №1 to watch anime online for free on Russian.",
+		"ru": "AnimeVost - База №1 для просмотра аниме онлайн бесплатно на Русском."
+	},
+	"url": [
+		"animevost.org",
+		"v2.vost.pw"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/9KbVqtj.png",
```

Comment:
Logos should be 512x512px

Reply:
This is not resolved yet

Reply:
But I've resized the Image to 512x512px and reuploaded it to imgur, then changed all the links to new one.

Reply:
Sry, now I see that I changed links in "presence.ts" but forgot to change link in "metadata.json". Will change it when Im gonna be at home.